syntax = "proto3";

package sonarData;  // Optional: Replace with your desired package name

// Define a message type for an array of points.
message sonarData {
    repeated float pointX = 1;
    repeated float pointY = 2;
    repeated float beamIdx = 3;
}


// To include new data, add it to this code and run "protoc --c_out=. data.proto" 
//      from the same folder as the data.proto is in
// For python: protoc --python_out=. sonarData.proto